src/v/config/configuration.cc

```diff
@@ -3792,6 +3792,23 @@ configuration::configuration()
       {.visibility = visibility::user},
       std::nullopt,
       &validate_non_empty_string_opt)
+  , iceberg_backlog_controller_p_coeff(
+      *this,
+      "iceberg_backlog_controller_p_coeff",
+      "Proportional coefficient for the Iceberg backlog controller. Number of "
+      "shares assigned to the datalake scheduling group will be proportional "
+      "to the backlog size error.",
```

Comment:
I think it's worth leaving a note around tuning this, like if more negative values mean that the system has more aggressive adaptivity to workloads or something